eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/log/EventLogWriter.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2015 - 2016 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.eventuate.log
+
+import akka.actor._
+
+import com.rbmhtechnology.eventuate._
+
+import scala.collection.immutable.Seq
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util._
+
+private object EventLogWriter {
+  class EventLogWriterActor(val id: String, val eventLog: ActorRef) extends EventsourcedActor {
```

Comment:
Something very similar exists here: `com.rbmhtechnology.eventuate.DeleteEventsSpecLeveldb.Emitter`. Maybe it makes sense to reuse and drop one or the other.